mattermost-9/ENTERPRISE-EDITION-LICENSE.txt

```diff
@@ -0,0 +1,6 @@
+The Mattermost Enterprise Edition (EE) license (the “EE License”)
```

Comment:
Why do we have this? It doesn't sound like we're building the enterprise edition.